plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/messages.properties

```diff
@@ -1 +1,2 @@
-AppEngineStandardWizardPage_librariesGroupLabel=Libraries to add to build path
+appengine.standard.wizard.page.libraries.group.label=Libraries to add to build path
+appengine.java.component.missing=An App Engine project cannot be created because the Cloud SDK App Engine Java component is not installed. Fix by running 'gcloud components install app-engine-java' on the command-line
```

Comment:
"An App Engine project cannot be created" --> "Cannot create an App Engine Eclipse project"